Resources/Prototypes/Entities/Objects/Specific/Mech/mecha_equipment.yml

```
@@ -69,4 +69,4 @@
   - type: UIFragment
     ui: !type:MechSoundboardUi
   - type: UseDelay
-    delay: 0.5
+    delay: 0.5
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)